book_source/source/module-9-miscellaneous-topics/lesson-27-reading-ethics-in-data-science/wrap-up-and-assignment.md

```diff
@@ -1,56 +1,21 @@
-
-A valid post is unique and reflects on what you learned and specifically talks about one of the concepts from the reading (i.e., demonstrates that you did the reading). This can be in the form of sharing your opinion on one of these case studies and whether or not it's ethical to use them, adding on to the considerations we shared (or objecting to any of the points we shared), or relating what you learned to some other application you have seen. **You do not need to talk about every case study in your response, it's okay to focus on one** ; you are more than welcome to relate different case studies though! Additionally, you can respond to someone else's post to add a different perspective or disagree with their point and elaborate on why; a debate is good, but **remember to be respectful** and make sure we are asking critical questions about what we believe! Do remember that you are just one voice and other people likely have different opinions than you. On top of your reflection or response to someone else's post, you are welcome to pose any additional discussion questions that people can follow up with!
-
-You should not merely state the same thing someone else in your quiz section says. That means before posting, you should read through the other responses in your quiz section's discussion thread to make sure what you are going to share is not just the same opinion as someone else.
-
-You don't need to spend a ton of time worrying if your post is enough; we want you to participate in the discussion and gain something new from your classmates' perspectives. As long as you put in a serious effort to meet our requirements and demonstrate that you did the reading, you will earn credit!
-
-###  Discussion Board Posts
-
-You should post in the thread for your quiz section. Make sure you check your quiz section on MyUW to make sure you are posting in the right one!
-
--  [None](https://edstem.org/us/courses/3016/discussion/289186)
-
--  [None](https://edstem.org/us/courses/3016/discussion/289187)
-
--  [None](https://edstem.org/us/courses/3016/discussion/289189)
-
--  [None](https://edstem.org/us/courses/3016/discussion/289196)
-
--  [None](https://edstem.org/us/courses/3016/discussion/289197)
-
--  [None](https://edstem.org/us/courses/3016/discussion/289198)
-
--  [None](https://edstem.org/us/courses/3016/discussion/289200)
 
--  [None](https://edstem.org/us/courses/3016/discussion/289203)
+## Assignment
 
+Your practice problem for today is to post on the course discussion board reflecting on what you learned today and sharing your opinion or experience relating to one of the topics addressed in this lesson.
```

Comment:
Just wondering if it is appropriate to mention the course discussion board here for high school curriculum.